lua/starfall/libs_cl/render.lua

```diff
@@ -323,6 +323,22 @@ end, function(instance)
 	return true
 end)
 
+--- Called before the 3D skybox is drawn. This will not be called for maps with no 3D skybox, or when the 3d skybox is disabled
+-- @name predrawskybox
+-- @class hook
+-- @client
+-- @return boolean Return true to not predraw the skybox both 2d and 3d
+SF.hookAdd("PreDrawSkyBox", nil, hudPrepareSafeArgs, function(instance, args)
+	instance:cleanupRender()
+    if args[1] and args[2]==true then return true end
```

Comment:
I don't think this should be allowed to return unless LocalPlayer() is the chip's owner

Reply:
Why though? People can already do stuff more crazy than this when connected to a HUD.

Reply:
I'm PeterPettigrew in discord, this is my first pullrequest, so dont expect it to be perfect

Reply:
also, my first rummage into actual lua instead of just starfall

Reply:
Nvm, I didn't realize that it was checking if a hud was connected